input/core/src/commonMain/kotlin/majestic/ToggleSwitch.kt

```diff
@@ -36,13 +37,16 @@ fun ToggleSwitch(
     backgroundOffColor: Color = Color(0xFFF2F4F7),
     circleOnColor: Color = Color.White,
     circleOffColor: Color = Color.White,
+    containerModifier: Modifier = Modifier
 ) {
+    Column(
```

Comment:
Why is this Column here? it looks like you are just adding it for some styling. Can't this Column be added at the call site? instead of the library side?

Reply:
container removed and moved to the upper layer